frontend/src/components/Tokens/TokenIcon.tsx

```diff
@@ -0,0 +1,43 @@
+import Icon, {IconProps} from "@chakra-ui/icon";
+import {QuestionIcon} from "@chakra-ui/icons";
+import { Flex } from "@chakra-ui/layout";
+
+// TODO this needs to be updated when new tokens are added
```

Comment:
Can we have a default image? So if someone forgets to, then it isn't an issue.
Alternatively,can this not be added in one of the constants.ts/constants.json files?

For enw tokens, all changes should be minimal i.e. ideally in just one file.